app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx

```diff
@@ -193,18 +195,19 @@ export const RuleGroup = ({ pluginName, typedRules, notifications, isPluginEnabl
   const onCopy = (notification) => {
     trackEvent(PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.CLICK_ICON_DUPLICATE_NOTIFICATIONS);
 
-    const { id, ...newNotification } = notification;
+    const { id, ...newNotification } = { ...notification, ...notification?.ruleDetails };
```

Comment:
I see that this logic with exposing `ruleFields` to `notification` repeated several times (`{ ...notification, ...notification?.ruleDetails }`), so does it make sense to create a separate function for this?
It can be named like `flatRule()` or smth like that.
What do you think?